docs/accounts-billing/my-organization.md

```diff
@@ -0,0 +1,287 @@
+---
+slug: my-organization
```

Comment:
Hey @pinkeshmars, I think this should be part of FF UI directory itself. Since this is significant for teams plan. Can we please move this?

Reply:
Sure! moved.